network/p2p/node/libp2pNode_test.go

```diff
@@ -319,7 +314,31 @@ func createConcurrentStreams(t *testing.T, ctx context.Context, nodes []p2p.LibP
 		// in 2 connections 1 created by each node, this happens because we are calling CreateStream concurrently.
 		time.Sleep(500 * time.Millisecond)
 	}
-	unittest.RequireReturnsBefore(t, wg.Wait, 3*time.Second, "could not create streams on time")
```

Comment:
I think this test started running faster, so this check would fail because it got here sooner and had to wait for the context timeout.